components/resources/InfiniteScrollResources.tsx

```diff
@@ -62,7 +62,10 @@ function InfiniteScroll({
 									{resource.name}
 								</div>
 								<div className='text-xs font-medium leading-6 text-gray-900'>
-									{`${resource.city}, ${resource.state}, ${resource.country_code}`}
+									{`${resource.city.length === 0
```

Comment:
Might be able to shorten this up with ``${resource.city.length > 0 && resource.city},`

Reply:
Thanks I will implement this when I get back home

Reply:
Took a look real quick because before I have to run but that returns "false" in place of where the city would go instead of the blank. I can look into more.

Reply:
Okay, I thought this might be the case. No worries, I'll merge now. Thanks!